pkg/audit/containment.go

```diff
@@ -19,9 +19,6 @@ var (
 	// ErrContainedNotFound is the errs class for when a pending audit isn't found
 	ErrContainedNotFound = errs.Class("pending audit not found")
 
-	// ErrContainDelete is the errs class for when a pending audit can't be deleted
```

Comment:
Perhaps the error should exist and the real issues is that it has not been used where it should be; have you checked that?

Reply:
good catch, I added this back and used it in satellitedb/containment.go